client/Android/Studio/freeRDPCore/build.gradle

```diff
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion = 27
-    buildToolsVersion = "27.0.3"
+    compileSdkVersion = 28
+    buildToolsVersion = "28.0.3"
 
     defaultConfig {
         minSdkVersion 14
```

Comment:
Might be a good idea to raise this to `21` too

Reply:
Good spotting. Done..